app.js

```diff
@@ -16,8 +17,10 @@ app.get('/api', (req, res) => {
 	res.status(200).send({ endpoints })
```

Comment:
Please only add code that is required for use in the branch. This is not required by your app.server as of yet so shouldn't be added. Please delete.